tests/test_domain_py.py

```diff
@@ -178,6 +180,10 @@ def test_resolve_xref_for_properties(app, status, warning):
             ' title="module_a.submodule.ModTopLevel.prop">'
             '<code class="xref py py-meth docutils literal notranslate"><span class="pre">'
             'prop</span> <span class="pre">method</span></code></a>' in content)
+    assert ('Link to <a class="reference internal" href="#module-a-submodule-modtoplevel-prop"'
+            ' title="module_a.submodule.ModTopLevel.prop">'
+            '<code class="xref py py-prop docutils literal notranslate"><span class="pre">'
```

Comment:
```suggestion
            '<code class="xref py py-attr py-prop docutils literal notranslate"><span class="pre">'
```

Backwards compatibility for themes without sacrificing the ability to style both differently.